bokehjs/src/less/panels.less

```diff
@@ -2,7 +2,8 @@
 
 :host {
   position: absolute;
-  background-color: white;
+  background-color: var(--background-color);
+  // @matpapp - why are these commented out?
```

Comment:
This should have been removed. Originally `Panel` was meant to be one thing, but ended up being a different kind of component, for which setting default borders and padding doesn't make sense.